config/application.rb

```diff
@@ -1,21 +1,29 @@
-require_relative 'boot'
+require_relative "boot"
 
 require "rails"
+# Pick the frameworks you want:
 require "active_model/railtie"
+require "active_job/railtie"
 require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
 require "action_view/railtie"
+# require "action_cable/engine"
 require "sprockets/railtie"
 require "rails/test_unit/railtie"
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-require_relative '../lib/bot_killer'
+require_relative "../lib/bot_killer"
 
 module RailsContributors
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
```

Comment:
I think we could use 6.1 here to keep it up to date.

Reply:
I'm planning to do that in another PR when I flip all the configs part of the new defaults. Some configs should not be changed in the same deploy that upgrade the framework.